web/Export/PhpECSExporter.ts

```diff
@@ -61,12 +61,12 @@ export default class PhpECSExporter implements ExporterInterface {
 
         if (configuration.fixerSets !== undefined) {
             lines.push(INDENT + '$parameters->set(Option::SETS, [');
-            lines.push(INDENT + INDENT + '// must be done manually, as names differ');
+            lines.push(INDENT + INDENT + '// @TODO must be reviewed manually, as names differ and there is no set-exclude in easy-coding-standard');
             configuration.fixerSets.forEach((fixerSetName: string): void => {
                 if (fixerSetName.charAt(0) === '-') {
                     lines.push(INDENT + INDENT + '// exclude: SetList::' + fixerSetName.substr(2) + ',');
```

Comment:
There's no way to implement this?

Reply:
What about https://github.com/ckrack/php-cs-fixer-configurator/pull/1 ?

Reply:
awesome!

Only way i can think of is to use the SKIP-option from ecs and name each rule individually with a comment why it was excluded, but I'd rather open a PR on ecs repo to get an exclude-set option.

```php
    $parameters->set(Option::SKIP, [
        // skip rule because it was excluded with @Symfony:risky
        IsNullFixer::class => null,
        …
    ]);
```